docs/data/sdks/android-kotlin/index.md

```diff
@@ -22,31 +22,92 @@ Use [this quickstart guide](../../sdks/sdk-quickstart#android) to get started wi
-    | `useBatch` |  Whether to use batch api. | `false` |
-    | `useAdvertisingIdForDeviceId` |  Whether to use advertising id as device id. Need to include the module and permission. | `false` |
-    | `useAppSetIdForDeviceId` |  Whether to use app ser id as device id. Need to include the module and permission. | `false` |
-    | `trackingOptions` |  Options to control the values tracked in SDK. | `enable` |
-    | `enableCoppaControl` |  Whether to enable coppa control for tracking options. | `false` |'
-
-### `track`
+    | `flushIntervalMillis` | `Int`. The amount of time SDK will attempt to upload the unsent events to the server or reach `flushQueueSize` threshold. | `30000` |
+    | `flushQueueSize` | `Int`. SDK will attempt to upload once unsent event count exceeds the event upload threshold or reach `flushIntervalMillis` interval.  | `30` |
+    | `flushMaxRetries` | `Int`. Maximum retry times.  | `5` |
+    | `minIdLength` | `Int`. The minimum length for user id or device id. | `5` |
+    | `partnerId` | `Int`. The partner id for partner integration. | `null` |
+    | `identifyBatchIntervalMillis` | `Long`. The amount of time SDK will attempt to batch intercepted identify events. | `30000` |
+    | `flushEventsOnClose` | `Boolean`. Flushing of unsent events on app close. | `true` |
+    | `callback` | `EventCallBack`. Callback function after event sent. | `null` |
+    | `optOut` | `Boolean`. Opt the user out of tracking. | `false` |
+    | `trackingSessionEvents` | `Boolean`. Flushing of unsent events on app close. | `false` |
+    | `minTimeBetweenSessionsMillis` | `Long`. The amount of time for session timeout if disable foreground tracking. | `300000` |
+    | `serverUrl` | `String`. The server url events upload to. | `https://api2.amplitude.com/2/httpapi` |
+    | `serverZone` | `ServerZone.US` or `ServerZone.EU`. The server zone to send to, will adjust server url based on this config. | `ServerZone.US` |
+    | `useBatch` | `Boolean` Whether to use batch api. | `false` |
+    | `useAdvertisingIdForDeviceId` | `Boolean`. Whether to use advertising id as device id. Need to include the module and permission. | `false` |
+    | `useAppSetIdForDeviceId` | `Boolean`.  Whether to use app ser id as device id. Need to include the module and permission. | `false` |
+    | `trackingOptions` | `TrackingOptions`. Options to control the values tracked in SDK. | `enable` |
+    | `enableCoppaControl` | `Boolean`. Whether to enable coppa control for tracking options. | `false` |'
+
+#### Configure batching behavior
+
+--8<-- "includes/sdk-ts/shared-batch-configuration.md"
+
+=== "Java"
+
+    ```java 
+    import com.amplitude.android.Amplitude;
+
+    Amplitude amplitude =  new Amplitude(new Configuration(
+        apiKey = AMPLITUDE_API_KEY,
+        context = applicationContext,
+        flushIntervalMillis = 50000,
+        flushQueueSize = 20,
```

Comment:
20 may be too less in this case? I know it is example though still prefer a larger number here as we are talking about  batch.